crypto/encoding/amino/amino.go

```diff
@@ -19,6 +23,23 @@ func init() {
 	// https://github.com/tendermint/go-amino/issues/9
 	// is resolved
 	RegisterAmino(cdc)
+
+	// TODO: Have amino provide a way to go from concrete struct to route directly.
+	// Its currently a private API
```

Comment:
If not happened already: please open an issue in amino too and link all relevant issues / PRs.